File: java/Maximum Number of Operations to Move Ones to the End.java
// Maximum Number of Operations to Move Ones to the End - 3228

class Solution {
    public int maxOperations(String s) {
        int ans = 0, cnt = 0;
        int n = s.length();
        for (int i = 0; i < n; ++i) {
            if (s.charAt(i) == '1') {
                ++cnt;
            } else if (i > 0 && s.charAt(i - 1) == '1') {
                ans += cnt;
            }
        }
        return ans;
    }
}

File: python/Maximum Number of Operations to Move Ones to the End.py
# Maximum Number of Operations to Move Ones to the End - 3228

class Solution:
    def maxOperations(self, s: str) -> int:
        ans = cnt = 0
        for i, c in enumerate(s):
            if c == "1":
                cnt += 1
            elif i and s[i - 1] == "1":
                ans += cnt
        return ans